compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -1867,6 +1867,9 @@ void OMR::X86::CodeGenerator::doRegisterAssignment(TR_RegisterKinds kindsToAssig
 
       self()->doBackwardsRegisterAssignment(kindsToAssign, self()->getAppendInstruction());
       }
+
+   if (TR::Options::getCmdLineOptions()->getOption(TR_EnableCodeCacheDisclaiming))
```

Comment:
Do you think this feature (moving the OOL code to the warm code area) should have its own enabling (or disabling) option or env var independent of whether code cache disclaiming is enabled?  If there is room for experimentation with this feature then maybe it should have a way of being disabled.

Reply:
Sure, I will add a separate option.

Reply:
done